bpmn-js-app/angular.json

```diff
@@ -133,5 +138,8 @@
       }
     }
   },
-  "defaultProject": "bpmn-js-angular"
+  "defaultProject": "bpmn-js-angular",
```

Comment:
What does this do? How does this contribute / improve to the example?

Reply:
@nikku: it was generated automatically after `ng update @angular/core`, because in Angular 9 you will be asked whether you wish to send analytics data to angular.io